.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+
+install:
+  - pip install -r requirements.txt
+  - git clone https://github.com/simphony/JYU-LB.git
```

Comment:
Could you specify what should be moved: only the git clone line, or all the lines between pip install and python setup.py? And is a simple cut-paste appropriate?